docs/source/reference/indexing.rst

```diff
@@ -40,6 +40,8 @@ Modifying and computations
    Index.is_interval
    Index.is_numeric
    Index.is_object
+   Index.min
+   Index.max
```

Comment:
@itholic can you create another PR to add all missing functions in `MultiIndex` below?

Reply:
@HyukjinKwon okay, i'll take it